assets/src/scripts/components/map/index.spec.js

```diff
@@ -138,4 +139,192 @@ describe('map module', () => {
             expect(select(mapNode).select('a#fim-link').node()).toBeNull();
         });
     });
+
+
+    describe('Map creation without NLDI maps', () => {
+        beforeEach(() => {
+            store = configureStore();
+            attachToNode(store, mapNode, {
+                siteno: '12345677',
+                latitude: 43.0,
+                longitude: -100.0,
+                zoom: 5
+            });
+        });
+
+        it('Should create a leaflet map within the mapNode with', () => {
+            expect(select(mapNode).selectAll('.leaflet-container').size()).toBe(1);
+        });
+
+        it('Should create a marker layer', () => {
+            expect(select(mapNode).selectAll('.leaflet-marker-pane img').size()).toBe(1);
+        });
+
+        it('Should not create NLDI layers', () => {
```

Comment:
Another nit. Maybe string should be 'Should not create an overlay layers'. The FIM layers are also in the overlay pane.

Reply:
Yup - good point, I'll reword this.